### PR TITLE
Update for move to new run notes db

### PR DESCRIPTION
**Pull request by aanil**
ONT related processes should write to new running notes db.

Dependent on https://github.com/SciLifeLab/StatusDB_views/pull/149

### CODE REVIEW

**Review by kedhammar — APPROVED** [review]
Old stuff looks nested and else-clause:d correctly to me :)

File: scripts/comments_to_running_notes.py [hunk 1]
```diff
@@ -158,10 +158,8 @@ def main(lims, args):
     noteobj = {}
     pro = Process(lims, id=args.pid)
     if "Comments" in pro.udf and pro.udf["Comments"] is not "":
-        key = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
-        noteobj[key] = {}
         if isinstance(pro.udf["Comments"], str):
-            comments = pro.udf["Comments"]
+                comments = pro.udf["Comments"]
```
Comment (alneberg) [inline scripts/comments_to_running_notes.py:162]: 8 space indentation is a bit too much isn't it?

Comment (alneberg) [inline scripts/comments_to_running_notes.py:162]:
```suggestion
            comments = pro.udf["Comments"]
```

**Review by alneberg — APPROVED** [review]
Looks good to me as well! Thank you!

**Review by chuan-wang — APPROVED** [review]

**Pull request merged**